src/webapi/event_target.rs

```diff
@@ -7,6 +7,7 @@ use webapi::event::{ConcreteEvent, IEvent};
 use private::UnimplementedException;
 
 /// A handle to a particular event listener.
+// TODO derive PartialEq, Eq, etc. ?
```

Comment:
It should be enough to compare `listener_reference` (since those are going to be unique for every `EventListenerHandle`) for `PartialEq` for this.

Reply:
The comment wasn't about *how* to implement `PartialEq`, it was about whether we should implement `PartialEq` *at all* on that type.